<launch>
    <node pkg="dev_pkg" type="soft_sync_node" name="soft_sync_node" output="screen">
        <param name="image_topic" value="/camera/color/image_raw"/>
        <param name="lidar_topic" value="/livox/lidar"/>
        <param name="synced_image_topic" value="/synced_image"/>
        <param name="synced_lidar_topic" value="/synced_lidar"/>
        <param name="publish_rate" value="10.0"/>
    </node>    
</launch>
